ledger/src/blockstore_processor.rs

```diff
@@ -3946,7 +3946,9 @@ pub mod tests {
         genesis_config.ticks_per_slot = TICKS_PER_SLOT;
```

Comment:
nit: I'd recommend to rename this test to `test_confirm_slot_entries_without_fix`

so that, git remembers that the existing test and newly-added test is logically-grouped.
also i like equal degree of qualification for logically parallel tests. i mean `test_confirm_slot_entries` (without `_without_fix`) sounds more general test than `test_confirm_slot_entries_with_fix`. in fact, they're logically parallel in that they're exposing deferring behavior before/after the feature activation.